oceannavigator/frontend/src/components/MapToolbar.jsx

```diff
@@ -312,11 +312,26 @@ class MapToolbar extends React.Component {
 
   // Instructs the OceanNavigator to fetch point data
   applyPointCoords() {
+    let latitude = this.state.coordinate[0]
+    let longitude = this.state.coordinate[1]
+    //Latitude ranges between -90 and 90 degrees, inclusive. 
+    //Values above or below this range will be clamped to the range [-90, 90]. 
+    //This means that if the value specified is less than -90, it will be set to -90. 
+    //And if the value is greater than 90, it will be set to 90.
+    latitude = (latitude >90) ? 90: latitude;
+    latitude = (latitude <-90) ? -90: latitude;
+    //Longitude ranges between -180 and 180 degrees, inclusive. 
+    //Values above or below this range will be wrapped so that they fall within the range.
```

Comment:
Rather than forcing the latitude to be +/- 90 when outside the range it might be better have the value wrap around the globe. For example a user input of 100 degrees latitude would be converted to 80 degrees, 200 to -20 (very unlikely but should be considered), etc.

Reply:
Actually I was using the concept from the below link:
https://developers.google.com/maps/documentation/javascript/reference/coordinates